k8s/gateway/gateway.yml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: gateway
+  labels:
+    app: gateway
+spec:
+  ports:
+    - name: http
+      port: 8098
+      targetPort: 8098
+      nodePort: 30009
+  selector:
+    app: gateway
+  type: LoadBalancer
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gateway
+  namespace: default
+  labels:
+    app: gateway
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gateway
+  template:
+    metadata:
+      labels:
+        app: gateway
+    spec:
+      containers:
+        - name: gateway
+          image: 'lansoftdev/gateway:0.0.1'
+          imagePullPolicy: Always
+          ports:
+            - name: http
+              containerPort: 8098
+              protocol: TCP
+          env: 
+            - name: DB_CLICKHOUSE_URL
+              value: jdbc:clickhouse://clickhouse:8123/default
+            - name: DB_CLICKHOUSE_USER
+              value: default
+            - name: DB_CLICKHOUSE_PASSWORD
+              value:
+            - name: DB_CLICKHOUSE_COMTRESS
+              value: 'true'
+            - name: KEYCLOAK_AUTHSERVERUML
+              value: http://172.28.128.3:30008/auth/
```

Comment:
Please, put a comment about this address. What I'd like to see here is why we use the address of VPC(VM in user case) instead of DNS name of service in k8s

Reply:
+ put such information into readme

Reply:
We can better. As this is an environment-specific value, let's move it into [configmap](https://kubernetes.io/docs/concepts/configuration/configmap/). Also, make sure to put a note about that in the readme.